cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SpatialTest.scala

```diff
@@ -273,7 +273,7 @@ class SpatialTest extends DocumentingTest {
     section("Spatial index", "cypher-spatial-index") {
```

Comment:
I'm not sure about this, but since we know have a spatial index called _point index_ maybe we should simply start using that name everywhere instead?

Reply:
I changed the mentions of spatial index to point index but did you want me to also change all references to spatial indexes to point or is it ok to keep it as is, since the point index is a spatial index?